instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/library/okhttp/v3_0/internal/OkHttp3Singletons.java

```diff
@@ -33,29 +31,33 @@ public final class OkHttp3Singletons {
     private static final Supplier<Instrumenter<Request, Response>> INSTRUMENTER =
             CachedSupplier.create(
                     () ->
-                            OkHttpInstrumenterFactory.create(
-                                    GlobalOpenTelemetry.get(),
-                                    builder ->
-                                            builder.setCapturedRequestHeaders(
-                                                            OkHttpInstrumentationConfig
-                                                                    .getCapturedRequestHeaders())
-                                                    .setCapturedResponseHeaders(
-                                                            OkHttpInstrumentationConfig
-                                                                    .getCapturedResponseHeaders())
-                                                    .setKnownMethods(
-                                                            OkHttpInstrumentationConfig
-                                                                    .getKnownMethods()),
-                                    spanNameExtractorConfigurer ->
-                                            spanNameExtractorConfigurer.setKnownMethods(
-                                                    OkHttpInstrumentationConfig.getKnownMethods()),
-                                    Function.identity(),
-                                    singletonList(
+                            OkHttpClientInstrumenterBuilderFactory.create(GlobalOpenTelemetry.get())
+                                    .setCapturedRequestHeaders(
+                                            OkHttpInstrumentationConfig.getCapturedRequestHeaders())
+                                    .setCapturedResponseHeaders(
+                                            OkHttpInstrumentationConfig
+                                                    .getCapturedResponseHeaders())
+                                    .setKnownMethods(OkHttpInstrumentationConfig.getKnownMethods())
+                                    // TODO: Do we really need to set the known methods on the span
```

Comment:
Now that I think about it better, I'm not sure if any of our users would ever need to change the known HTTP method names overall... I guess it's there because there's a setter for it in the instrumentation lib, though apart from that, I don't think there's any reason to override the names either for the inst builder or for the span name extractor. 🤔